test/www/jxcore/test/thalicryptomanagertests.js

```diff
@@ -0,0 +1,61 @@
+require('./mockmobilegetdocpath');
+var test = require('tape');
+var cryptomanager = require('../thali/thalicryptomanager');
```

Comment:
This is going to break when you merge with story 0 because we no longer include thali via file path, instead its now a normal NPM dependency. So this path should become 'thali/thalicryptomanager'